java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -474,4 +474,21 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Double[]> zmscore(String key, String[] members);
+
+    /**
+     * Calculates the difference between the first sorted set and all the successive sorted sets at
+     * <code>keys</code> and stores the difference as a sorted set to <code>destination</code>,
+     * overwriting it if it already exists. Non-existent keys are treated as empty sets.
+     *
+     * @see <a href="https://redis.io/commands/zdiffstore/">redis.io</a> for more details.
+     * @param destination Destination key for the resulting sorted set.
```

Comment:
Maybe
```suggestion
     * @param destination A key for the resulting sorted set.
```
(update in transaction too)

Reply:
```suggestion
     * @param destinationKey The resulting sorted set is stored at this key.
```

Reply:
I have changed it to The key for the resulting sorted set.